fluent/src/resolver.js

```diff
@@ -294,7 +294,8 @@ function Type(env, expr) {
     }
     case undefined: {
       // If it's a node with a value, resolve the value.
-      if (expr.val !== undefined) {
+      // eslint-disable-next-line eqeqeq
+      if (expr.val != null) {
```

Comment:
why `!=` instead of `!==`?

Reply:
To check both for `null` and `undefined`; see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness.

Reply:
Just to make it clear in case it came across wrong: I don't suppose you don't know the difference :) Just wanted to link to the `Loose comparison` matrix for reference of what `!= null` evaluates to.

Reply:
since we usually use strict comparison, can you add a comment on why we use a different strategy here?

Reply:
I'll just use the more explicit double comparison. Can't hurt the readability. Thanks!